webapp/portlet/src/main/webapp/vue-apps/profile-header/components/ProfileHeader.vue

```diff
@@ -146,8 +146,10 @@ export default {
     document.addEventListener('userModified', event => {
       if (event && event.detail && event.detail !== this.user) {
         this.user = Object.assign({}, this.user, event.detail);
-        this.$nextTick().then(() => this.$root.$emit('application-loaded'));
+      } else  {
```

Comment:
This is a forcing to reload that is not needed. Could you check why the userModified is null ?

Reply:
the event is dispatched by ProfileContactInformationDrawer which has only the user's properties to modify. 
https://github.com/Meeds-io/social/blob/7f85b69e9c7edcbbfed6485087f05eee94c865f7/webapp/portlet/src/main/webapp/vue-apps/profile-contact-information/components/ProfileContactInformationDrawer.vue#L205

Reply:
FYI @rdenarie

Reply:
@GouadriaHanen the check inside the event listener should be updated without the need to call refresh explicitly which will reload data from server.
you can convert the array of properties in https://github.com/Meeds-io/social/blob/7f85b69e9c7edcbbfed6485087f05eee94c865f7/webapp/portlet/src/main/webapp/vue-apps/profile-contact-information/components/ProfileContactInformationDrawer.vue#L205 to a user object  then send it in the event data . this way you won't need to change in [ProfileHeader.vue](https://github.com/Meeds-io/social/pull/2353/files#diff-8a7df92ac03c601f3d8d0748d9960ec49280076846328494cb5c16bfe7b57256)

Reply:
done!